Mage/src/main/java/mage/abilities/hint/common/PermanentsSacrifiedThisTurnHint.java

```diff
@@ -0,0 +1,28 @@
+package mage.abilities.hint.common;
+
+import mage.abilities.Ability;
+import mage.abilities.dynamicvalue.common.PermanentsSacrificedThisTurnCount;
+import mage.abilities.hint.Hint;
+import mage.abilities.hint.ValueHint;
+import mage.game.Game;
+
+/**
+ * @author Susucr
+ */
+public enum PermanentsSacrifiedThisTurnHint implements Hint {
+    instance;
+
+    private static final Hint hint = new ValueHint(
+            "Permanents sacrified this turn", PermanentsSacrificedThisTurnCount.instance
```

Comment:
typo- sacrificed

Reply:
Oops, pardon my mistranslated french, we do write "sacrificed" as "sacrifié".